.github/workflows/build.yaml

```
@@ -7,11 +7,13 @@ on:
   build:
-    name: Create Release
+    name: Create keycloak-theme.jar
     runs-on: ubuntu-latest
     strategy:
       matrix:
         node-version: [18.18.2]
+    outputs:
+      version: ${{ steps.version.outputs.version }}
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)